### PR TITLE
Lightwave registration/de-registration support.

### PR DESCRIPTION
**Pull request by GeoffAtHome**
**Description:**
Lightwave registration/de-registration support.

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Your fork branch is to old, I see commits that already has been done and "live" in the current branch or revert changes that is not desirable.

**Comment by GeoffAtHome** [comment]
@klaasnicolaas the change I made was to update the file lightwave.markdown. It was not to revert any changes. I am probably not using Git correctly.  Please advise how to correct this?

**Comment by klaasnicolaas** [comment]
I understand that, but what I suspect is that you did not update your branch of your fork in the meantime, so that many older commits now work as a revert on the current situation.

File: source/_components/lightwave.markdown [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      name: Light socket
+    R2D3:
+      name: Phone socket
+    R2D4:
+      name: Torch socket
+    registration:
+      name: Registration
+```
+### `host`
+Where `192.168.1.2` is the ip address of your Lightwave hub.
```
Comment (klaasnicolaas) [inline source/_components/lightwave.markdown:51]: is this as example?

Comment (GeoffAtHome) [reply source/_components/lightwave.markdown:51]: @klaasnicolaas I am going to try again as I want to get this right. I think the steps I need are:
1. fork
2. create a branch
3. do the work
4. commit
5. Raise the PR

Also done on this now.

**Pull request closed without merging**